osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -68,13 +70,20 @@ private void load()
 
         protected abstract Drawable[] CreateAdditionalContent(TModel item);
 
-        protected override Drawable CreateHeader(int index, TableColumn column) => new HeaderText(column?.Header ?? string.Empty, HighlightedColumn());
+        protected override Drawable CreateHeader(int index, TableColumn column)
+        {
+            var title = column?.Header ?? string.Empty;
+            var isHighlighted = HighlightedColumn() == title;
+            var isGrade = GradeColumns().Contains(title);
+
+            return new HeaderText(title, isHighlighted) { Margin = new MarginPadding { Vertical = 5, Horizontal = isGrade ? 20 : 10 } };
```

Comment:
This long line is too long, when it could be split across object initialiser braces.